css-three-column-layout/styles.css

```diff
@@ -0,0 +1,71 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  padding: 0;
+  background-color: #05a8f3;
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: "Lato", sans-serif;
```

Comment:
If you remove the quotes, you won't need this stylelint directive

Reply:
Updated